dnf-behave-tests/features/steps/repo.py

```diff
@@ -261,6 +261,7 @@ def given_package_version(context, package):
 
 
 @behave.step("{quantifier} HTTP {command} request should match")
+@behave.step("{quantifier} HTTP {command} requests should match")
```

Comment:
Is this intentional? Does it still work?

Reply:
Yes, this is intentional (see the step `Then exactly 48 HTTP GET requests should match:`)

Reply:
Okay, but what about the tests that use the singular form? Is it something behave handles transparently? I suppose it has to be so...

Reply:
It indeed does. TBH, I haven't found any mention of this in the behave docs, but it works. I agree it's not the most elegant way, but I haven't found a better one.

Reply:
Oh, I just might have found a better one:
https://stackoverflow.com/a/53614851

Will take a look.

Reply:
FWIW I'm fine with the way it is... it might confuse someone and may be hard to grep (when someone say greps for "request should match" to look up the step definition, they won't find it), but then the form with regexes will as well.

As long as it works...

Reply:
OK, let's keep it as it is then.